os/services/ip64/ip64-arp.c

```diff
@@ -285,7 +285,7 @@ ip64_arp_check_cache(const uint8_t *nlhdr)
     /* Multicast. */
     return 1;
   } else {
-    uip_ip4addr_t ipaddr;
+    uip_ip4addr_t ipaddr = {0};
```

Comment:
I don't think we need to initialize `ipaddr` in this line - it is never used before a value is assigned to it later.

Reply:
Agreed, I will remove (I guess this is not right place for debate on initializing variables 😄 )